### ISSUE TITLE
Hi, I would like to ask how to generate labels with segmentation tool. kitti-360 dataset generates labels with mmsegmentation and then the file format is in pgm format is this correct? Thank you for your answer.

### ISSUE THREAD

**Issue by sxy971118** [video2game/video2game#6]
Hi, I would like to ask how to generate labels with segmentation tool. kitti-360 dataset generates labels with mmsegmentation and then the file format is in pgm format is this correct? Thank you for your answer.

**Comment by xiahongchi** [comment]
yes, it's correct. you can see here: https://github.com/video2game/video2game/blob/7bcf8ae754240b9fd2de75a154eb73c75a146b79/datasets/kitti360.py#L578

**Comment by sxy971118** [comment]
> 是的，这是正确的。你可以在这里看到：
> 
> https://github.com/video2game/video2game/blob/7bcf8ae754240b9fd2de75a154eb73c75a146b79/datasets/kitti360.py#L578

Thanks for the information. Can I ask again what segmentation tools were used to process the instance file in the Gardenvase scene? Thank you very much have a great day!

**Comment by xiahongchi** [comment]
Hi, I think we only need instance label in ".npy" file with shape of H, W.

You could look here:

https://github.com/video2game/video2game/blob/7bcf8ae754240b9fd2de75a154eb73c75a146b79/datasets/colmap.py#L232

https://github.com/video2game/video2game/blob/7bcf8ae754240b9fd2de75a154eb73c75a146b79/datasets/colmap.py#L241

https://github.com/video2game/video2game/blob/7bcf8ae754240b9fd2de75a154eb73c75a146b79/datasets/colmap.py#L347

https://github.com/video2game/video2game/blob/7bcf8ae754240b9fd2de75a154eb73c75a146b79/datasets/color_utils.py#L65

**Comment by sxy971118** [comment]
Hi, sorry for asking a lot of questions so I feel like I'll be bothering you. how should kitti360.py be run? What should I do if I need the read_semantics function to process the data? Do I need to write a script myself? When I wrote my own today frame_id first got [1.0000e+00 9.0000e+00 1.4000e+01 ... 1.1496e+04 1.1498e+04 1.1501e+04]
But after that it becomes frame_id: [] 。Still don't seem to know how to proceed when dealing with dataset instances.Hope you have a great day today.

**Comment by sxy971118** [comment]
Hi, sorry for asking a lot of questions so I feel like I'll be bothering you. how should kitti360.py be run? What should I do if I need the read_semantics function to process the data? Do I need to write a script myself? When I wrote my own today frame_id first got [1.0000e+00 9.0000e+00 1.4000e+01 ... 1.1496e+04 1.1498e+04 1.1501e+04]
But after that it becomes frame_id: [] 。Still don't seem to know how to proceed when dealing with dataset instances.Hope you have a great day today.

**Comment by sxy971118** [comment]
> yes, it's correct. you can see here:
> 
> https://github.com/video2game/video2game/blob/7bcf8ae754240b9fd2de75a154eb73c75a146b79/datasets/kitti360.py#L578

Hi, sorry for asking a lot of questions so I feel like I'll be bothering you. how should kitti360.py and colmap.py be run? What should I do if I need the read_semantics function to process the data? Do I need to write a script myself? When I wrote my own today frame_id first got [1.0000e+00 9.0000e+00 1.4000e+01 ... 1.1496e+04 1.1498e+04 1.1501e+04]
But after that it becomes frame_id: [] 。Still don't seem to know how to proceed when dealing with dataset instances.Hope you have a great day today.
